openapi-specs/cspm/Reports.json

```diff
@@ -973,7 +973,7 @@
       },
       "post": {
         "deprecated": true,
-        "description": "Creates a compliance report generation configuration based on the specified parameters. Report generation can be either one-time or recurring.\n:::info\n **Replacement  Endpoint: [Add Report Config V2](/prisma-cloud/api/cspm/save-report-v-2)**\n:::\n\n  \r\n\r\nYou can use the body parameters to specify whether the report is a one-time report \nor a recurring report. Specify a recurring report by providing a valid \n**target.schedule** body parameter.\n **Note:** `complianceStandardIds` is not applicable to Report Types `COMPLIANCE` or `RIS`\n",
+        "description": "Creates a compliance report generation configuration based on the specified parameters. Report generation can be either one-time or recurring.\n:::info\n **Replacement  Endpoint: [Add Report Config V2](/prisma-cloud/api/cspm/save-report-v-2)**\n:::\n\n  \r\n\r\nYou can use the body parameters to specify whether the report is a one-time report \nor a recurring report. Specify a recurring report by providing a valid \n**target.schedule** body parameter.\n\n**Note:** `complianceStandardIds` is not applicable to Report Types `COMPLIANCE` or `RIS`\n",
```

Comment:
Is this description for /v2/report or /report endpoint? If this description applies to /v2/report, please move the description to the correct endpoint.

Reply:
A minor edit to the description:  

The `complianceStandardIds` parameter is not applicable to `COMPLIANCE` and `RIS` report types.

Reply:
updated with latest commit! Thanks for catching that